messages/hww.proto

```diff
@@ -264,6 +264,25 @@ message RestoreFromMnemonicRequest {
   int32 timezone_offset = 2;
 }
 
+message DisplayBase32Request {
```

Comment:
Make one `oneof` request/response type for all Base things, same as ETHRequest and ETHResponse. Then it is better contained and easier to read/change/maintain.

Reply:
ack